benchmarking/README.md

```diff
@@ -51,6 +51,26 @@ $ ./preprocess.sh ~/mittelmann_benchmark ./mittelmann_instance_list \
     ~/mittelmann_preprocessed ~/PaPILO/build/bin/papilo
 ```
 
+## L1 SVM
+
+The L1 SVM instances apply the formulation from equation (5) in "1-norm Support
+Vector Machines" by J. Zhu et al. (NIPS, 2003.
+https://papers.nips.cc/paper/2003/file/49d4b2faeb4b7b9e745775793141e2b2-Paper.pdf).
+
+1. From the local directory, instantiate the necessary packages by running
+   `julia --project=. -e 'import Pkg; Pkg.instantiate()'`.
+2. Download the benchmark instances. Either run
```

Comment:
Editing fragment.  Presumably you meant either to add the same "or follow its steps..." from the Mittelmann section, or change "Either run" to "Run" and delete the space before the period at the end of the sentence.

Reply:
Oops! Fixed.